eve/workers/pod-example-solution-builder/Dockerfile

```diff
@@ -5,25 +5,25 @@ ARG OPERATOR_SDK_VERSION=v0.16.0
 
 WORKDIR /home/eve/workspace
 
-RUN yum install -y \
+RUN yum install -y --setopt=skip_missing_names_on_install=False \
     epel-release \
-    https://centos7.iuscommunity.org/ius-release.rpm \
-    && yum-config-manager \
-    --add-repo \
+    https://repo.ius.io/ius-release-el7.rpm \
+    && yum-config-manager --add-repo \
     https://download.docker.com/linux/centos/docker-ce.repo \
-    && yum install -y \
+    && yum install -y --setopt=skip_missing_names_on_install=False \
     sudo \
     gcc \
     hardlinks \
     make \
     genisoimage \
-    git2u \
     golang \
     python-devel \
     python-pip \
     skopeo \
     yum-utils \
+    yum-plugin-replace \
     docker-ce-cli-18.09.6 \
+    && yum replace -y git --replace-with git224 \
```

Comment:
Seems the `IUS` repo does not have the `git` package at all...

And if we must pin a git version may we just use `git224-all`?
Source: https://repo.ius.io/7/x86_64/packages/g/

Reply:
No, git package comes from CentOS repositories.
That's basically what's done here.